src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,34 @@
+.stars {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+
+  &__rate {
+    display: flex;
+
+    &-star {
+      display: flex;
+      height: 16px;
+      width: 16px;
+      background-image: url(../images/star.svg);
+      background-position: center;
+      background-repeat: no-repeat;
+    }
+
+    &-star:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &__rate div:nth-child(-n + 4) {
```

Comment:
The selector `div:nth-child(-n + 4)` is used to set the background image for the first four stars. If the number of active stars needs to be dynamic, consider using a more flexible approach, such as adding a class to active stars or using JavaScript to dynamically set the number of active stars.